docs/source/checks/vision/train_test_validation/plot_image_dataset_drift.py

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
 """
+.. _plot_vision_image_dataset_drift:
+
 Image Dataset Drift
-*********************
+*******************
```

Comment:
btw - quite sure too long underlines aren't problematic, only too short (no?)

Reply:
You're right. but it looks more beautiful now :)